docs/templates/language-reference.md

```diff
@@ -44,20 +44,45 @@ Ideally, also with a full working example that shows it in context.
 
 Depending on the language, a brief description of the recommended way to handle dependencies in the language.
 
-## Connect to services
+## Connecting to services
```

Comment:
:lady_beetle: This change does not fit the style guide: https://developers.google.com/style/headings?hl=en#heading-and-title-text